app/views/advanced_messenger/_body_bottom.html.erb

```diff
@@ -0,0 +1,170 @@
+<%# ============= My Page notifications ============= %>
+<% if current_page?(:controller => 'my', :action => 'page') %>
+    <%= javascript_tag do %>
+        $("#my-page").before(
+            "<div id='notifications-placeholder'></div>"
+        );
+    <%end%>
+    <%issues_or_forum_data = [{
+                                grouped_unread_notifications: getUnreadNotificationsGroupByIssues(),
+                                get_link: lambda {|issue| return "/issues/" + issue.id.to_s},
+                                my_page_all_notifications_read_text: :my_page_all_issues_notifications_read,
+                                my_page_unread_notifications_text: :my_page_issues_unread_notifications,
+                            }, 
+                            {
+                                grouped_unread_notifications: getUnreadNotificationsGroupByTopic(),
+                                get_link: lambda {|topic| return "/boards/" + topic.board.id.to_s + "/topics/" + topic.id.to_s },
+                                my_page_all_notifications_read_text: :my_page_all_forum_notifications_read,
+                                my_page_unread_notifications_text: :my_page_forum_unread_notifications
+                            }]
+    %>
+
+    <%issues_or_forum_data.each do |issues_or_forum| %> 
+        <% if issues_or_forum[:grouped_unread_notifications].count == 0%>
+            <%= javascript_tag do %>
+                $("#notifications-placeholder").append(
+                    "<h3 style='color:#2A5685'><%=t(issues_or_forum[:my_page_all_notifications_read_text])%></h3>"
+                );
+            <%end%>
+        <%else%>
+            <%= javascript_tag do %>
+                html = "<h3 style='color:#2A5685'><%=t(issues_or_forum[:my_page_unread_notifications_text])%></h3><ul style='list-style-type: none; padding-inline-start: 15px; font-weight: bold;'>";
```

Comment:
In the database for a topic (post) we have parent_id = null
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/41e3cc39-beb8-496b-9927-a68fda5c2670)

But in Rails, the ```root``` property gets computed for all messages (no matter is a topic or a reply).
Because in the interface they are rendered a litlle bit different I used ```if @message.id == @message.root.id``` condition to identify the topic among all the messages